### PR TITLE
Make native lib optional in OSGi environments

### PR DESCRIPTION
**Pull request by ibauersachs**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -245,7 +245,7 @@
         <dependency>
             <groupId>org.jitsi</groupId>
             <artifactId>jitsi-utils</artifactId>
-            <version>1.0-45-gc3afb76</version>
+            <version>1.0-53-g8535a53</version>
```
Comment (JonathanLennox) [inline pom.xml:248]: Is this necessary for the change, or a separate cleanup?

Comment (ibauersachs) [reply pom.xml:248]: Proably, I had -52 lying around for a while. But I can't compare it as I can't find these SHA1s. Is that built with a dirty tree on Jenkins?

Comment (JonathanLennox) [reply pom.xml:248]: No, the latest commit in jitsi-utils master is 8535a53 -- https://github.com/jitsi/jitsi-utils/commit/8535a5308592416852b5f5a974d92a7ef9d463d0 .

Comment (ibauersachs) [reply pom.xml:248]: Ok, I got confused by the stupidity of Git injecting a `g` in front of the actual hash.

And to your original question: yes the update is necessary to reference an OSGi enabled jitsi-utils.

**Pull request merged**